Lombiq.OrchardCoreApiClient/ConfigurableCertificateValidatingHttpClientHandler.cs

```diff
@@ -73,20 +73,16 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                         ["client_secret"] = _apiClientSettings.ClientSecret,
                     });
 
-            tokenResponse.ResponseMessage.EnsureSuccessStatusCode();
-
-            if (tokenResponse.GetContent().Error != null)
+            if (tokenResponse.Error is { } error)
```

Comment:
At least when there's no OC app running, and thus the token request will time out (with System.Net.Http.HttpRequestException: 'No connection could be made because the target machine actively refused it. (localhost:44335)'), instead of this, an exception will be thrown above, which will go unhandled.

Reply:
That exception wasn't handled in the original RestEase version either (I got that multiple times at the start because the app was running on port 5001 instead of the then hard coded 44335) so I assumed that was intentional.

Reply:
Actually the `tokenResponse.ResponseMessage.EnsureSuccessStatusCode();` in the original is exactly for that, see [the docs](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpresponsemessage.ensuresuccessstatuscode?view=net-7.0#exceptions), so that's the "normal" behavior. What do? Should I put a try around the API call and throw an `ApiClientException` if anything is caught?

Reply:
Yes, please.